include/inviwo/core/datastructures/geometry/mesh.h

```diff
@@ -228,6 +230,11 @@ IVW_CORE_API bool hasRadiiBuffer(const Mesh* mesh);
 
 }  // namespace meshutil
 
+
+// https://docs.microsoft.com/en-us/cpp/cpp/general-rules-and-limitations?view=vs-2017
+IVW_CORE_EXT template class IVW_CORE_API DataReaderType<Mesh>;
```

Comment:
An option would be to place these in separate files

Reply:
No that will NOT work, since we have to make sure the the client code sees this.